src/snowflake/snowpark/stored_procedure.py

```diff
@@ -68,6 +74,9 @@ def __init__(
         execute_as: typing.Literal["caller", "owner"] = "owner",
         anonymous_sp_sql: Optional[str] = None,
         packages: Optional[List[Union[str, ModuleType]]] = None,
+        _ast: Optional[proto.StoredProcedure] = None,
+        _ast_id: Optional[int] = None,
+        _stmt: Optional[proto.Assign] = None,
```

Comment:
should this be called `_ast_stmt` as done it some other places? Or is it different?